### PR TITLE
Fix documentation bugs. Fix duplicate reference.

### PR DESCRIPTION
**Pull request by ameltzerMSFT**
Fixes:
- #174 
- #171

### CODE REVIEW

File: src/WinGet.RestSource.Utils/WinGet.RestSource.Utils.csproj [hunk 1]
```diff
@@ -44,7 +44,6 @@
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.22.0" />
     <PackageReference Include="Microsoft.WindowsPackageManager.Utils" Version="0.3.4" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.12" />
-    <PackageReference Include="Microsoft.WindowsPackageManager.Utils" Version="*" />
```
Comment (yao-msft) [inline src/WinGet.RestSource.Utils/WinGet.RestSource.Utils.csproj:47]: "*" is actually preferred over specific version for Utils package so we can get the latest winget manifest object model. We could remove line 45 instead.

Comment (ameltzerMSFT) [reply src/WinGet.RestSource.Utils/WinGet.RestSource.Utils.csproj:47]: FIxed

**Review by yao-msft — APPROVED** [review]

**Comment by yao-msft** [comment]
Thanks for fixing!

**Pull request merged**